src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -101,6 +105,12 @@ public static class ReturnCode
                 public const int RetryableError = 2;
                 public const int AgentUpdating = 3;
             }
+
+            public enum AgentConfigurationProvider
+            {
+                AutomationAgentConfiguration,
```

Comment:
Call it BuildReleasesAgentConfiguration, we never use AutomationAgent in public doc.